packages/synapse-react-client/src/components/EntityCitation/EntityCitation.tsx

```diff
@@ -0,0 +1,205 @@
+  const { data: projectDoiAssociation } = useGetDOIAssociation(
+    {
+      id: projectId,
+      version: undefined,
+      type: DoiObjectType.ENTITY,
+    },
+    {
+      enabled: !!projectId,
+    },
+  )
+
+  // If both project and entity have DOIs, we show the dropdown variant
+  const isProjectAndEntityDoi = projectDoiAssociation && entityDoiAssociation
+
+  function getButtonText() {
+    if (isProjectAndEntityDoi) {
+      return 'Cite as...'
+    } else if (entityDoiAssociation && !projectDoiAssociation) {
+      return 'Cite page'
+    } else if (!entityDoiAssociation && projectDoiAssociation) {
+      return 'Cite project'
+    }
+    return ''
+  }
+
+  const buttonText = getButtonText()
+
+  const handleButtonClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    if (entityDoiAssociation && !projectDoiAssociation) {
+      setSelectedDoi(entityDoiAssociation.doiUri)
+      setCitationAnchorEl(event.currentTarget)
+    } else if (!entityDoiAssociation && projectDoiAssociation) {
+      setSelectedDoi(projectDoiAssociation.doiUri)
+      setCitationAnchorEl(event.currentTarget)
+    } else {
+      // If both project and entity doi exist, show dropdown
+      setAnchorEl(event.currentTarget)
+    }
+  }
+
+  const handleSelectCitation = (doi: string) => {
+    setSelectedDoi(doi)
+    setAnchorEl(null)
+    setCitationAnchorEl(anchorEl)
+  }
+
+  const handleClose = () => {
+    setAnchorEl(null)
+  }
+
+  const popover = (
+    <CitationPopoverContent
+      doi={selectedDoi}
+      anchorEl={citationAnchorEl}
+      open={!!citationAnchorEl && !!selectedDoi}
+      id={id}
+      onClose={() => {
+        setCitationAnchorEl(null)
+        setSelectedDoi(undefined)
+      }}
+    />
+  )
+
+  const dropdown = (
+    <Popover
+      open={open}
+      anchorEl={anchorEl}
+      onClose={handleClose}
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'left',
+      }}
+      transformOrigin={{
+        vertical: 'top',
+        horizontal: 'left',
+      }}
+      PaperProps={{
+        sx: {
+          padding: '10px 0',
+          borderRadius: '0px',
+          boxShadow: '0px 8px 24px 0px rgba(53, 58, 63, 0.15)',
+        },
+      }}
+    >
+      <MenuItem
+        value={projectDoiAssociation?.doiUri}
+        onClick={() => handleSelectCitation(projectDoiAssociation?.doiUri!)}
+      >
+        Cite this Project
+      </MenuItem>
+      <MenuItem
+        value={entityDoiAssociation?.doiUri}
+        onClick={() => handleSelectCitation(entityDoiAssociation?.doiUri!)}
+      >
+        Cite only this page
+      </MenuItem>
+    </Popover>
+  )
+
+  return (
+    <>
+      {(projectDoiAssociation || entityDoiAssociation) && (
+        <>
+          <Button
+            onClick={handleButtonClick}
+            sx={{
+              '& .MuiButton-endIcon': {
+                margin: 0,
+              },
+              borderColor: '#FFF !important',
+              color: '#FFF !important',
```

Comment:
This shouldn't be necessary--could this be removed completely? Or at least something like

```suggestion
              borderColor: 'currentColor',
              color: 'currentColor',
```

Reply:
Removed